src/Command/GenerateIdCommand.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Flex\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class GenerateIdCommand extends Command
+{
+    private $flex;
+
+    public function __construct(/* cannot be type-hinted */ $flex)
```

Comment:
Excuse me for asking (as it's already merged), but why it cannot be type-hinted? I'm curious.